tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -139,11 +140,24 @@ func {{ template "insertFunctionName" $schema }}(ctx context.Context, batch *pgx
         {{- range $field := $schema.DBColumnFields -}}
         {{- if eq $field.DataType "datetime" }}
         pgutils.NilOrTime({{$field.Getter "obj"}}),
+        {{- else if eq $field.SQLType "uuid" }}
+        pgutils.NilOrUUID({{$field.Getter "obj"}}),
```

Comment:
You may want to add the `uuid` (`github.com/stackrox/rox/pkg/uuid`) package to the import list because return type belongs to the uuid package.